client/app/components/calendar/layout/SideCalendar.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import moment from 'moment';
+import Calendar from 'rc-calendar';
+import bindAll from 'lodash/bindAll';
+import 'rc-calendar/assets/index.css';
+import './side_calendar.scss';
+
+export default class SideCalendar extends React.Component {
+  static contextTypes = {
+    router: React.PropTypes.object
+  };
+
+  constructor(...args) {
+    super(...args);
+    this._setMondayAsFirstWeekDay();
+    this.state = { value: moment() };
+
+    bindAll(this, ['onSelect', 'updateSelectedDate']);
+  }
+
+  onSelect(date) {
+    this._routeToSelectedDate(date);
+    this.updateSelectedDate(date);
+  }
+
+  updateSelectedDate(date) {
+    this.setState({ value: moment(date) });
+  }
+
+  disableWeekend(date) {
+    return date.day() === 0 || date.day() === 6;
+  }
+
+  render() {
+    return (
+      <Calendar value={this.state.value}
+                className="side-calendar"
+                showDateInput={false}
+                disabledDate={this.disableWeekend}
+                onSelect={this.onSelect}
+                onChange={this.updateSelectedDate} />
+    );
+  }
+
+  _setMondayAsFirstWeekDay() {
+    moment.updateLocale('en', {
+      week: {
+        dow: 1
+      }
+    });
```

Comment:
This doesn't seem like a good place to put global locale initialization.
Maybe put it in `Register.jsx` ? Or separate config file that you load in `Register.jsx`?